### PR TITLE
Bug fix/display reviews

### PR DESCRIPTION
**Pull request by aominhlong**
# Description
Please include a summary of the changes and/or which issue is fixed. Please also include relevant motivation and context. 
This PR adds refetch functionality to the App component in `filterBreweryReviews` to refetch and repopulate the reviews in state. This PR also will remove any commented out or unused code blocks. 

Were there any issues that arose? 
On page refresh, the `reviewsByBrewery` in State got wiped and defaulted by to an empty array. 

Is there anything you need from your teammates? 
N/A

Any other comments, questions, or concerns?
N/A

## Issues
Closes #

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [x] React Dev Tools
- [x] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have deleted all comments after functionality and understanding is final.
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: src/Components/App.tsx [hunk 1]
```diff
@@ -61,35 +83,60 @@ class App extends React.Component<{}, State> {
...
+      if (!this.state.reviews.includes(review)) {
+        this.setState({ reviews: [...this.state.reviews, review] })
+      }
+    })
+    filteredReviews.forEach(review => {
+      if (!this.state.reviewsByBrewery.includes(review)) {
+        this.setState({ reviewsByBrewery: [...this.state.reviewsByBrewery, review] })
+      }
+    })
+  }
```
Comment (rachellhill) [inline src/Components/App.tsx:120]: Nice workaround!

**Review by userigorgithub — APPROVED** [review]

**Review by rachellhill — APPROVED** [review]
Looks great! Nice job

File: src/Components/App.tsx [hunk 2]
```diff
@@ -44,12 +47,31 @@ class App extends React.Component<{}, State> {
...
+    return fetch(`http://localhost:3001/api/v1/reviews`)
+      .then(response => {
+          if (response.ok) {
+              return response.json()
+          } else {
+              throw Error(response.statusText)
+          }
+      })
+      .catch(error => console.log("error"))
+
```
Comment (stephanie-roe) [inline src/Components/App.tsx:74]: good call adding the refetch here!!

**Review by stephanie-roe — APPROVED** [review]
Looking great!

**Pull request merged**